csharp/ql/integration-tests/all-platforms/dotnet_run/test.py

```diff
@@ -0,0 +1,17 @@
+from create_database_utils import *
```

Comment:
Would it make sense to also expect the output of running the program (alternatively, I can do that as a follow up - which I dont' mind doing now that you have done all the heavy lifting 😂 ).
One thing is that the database is created, but unless we inspect the program output we haven't verified that the arguments has been re-written as we expect.

Something along the lines of:
```python
import os
from create_database_utils import *

def run_codeql_database_create_stdout(args, dbname):
    stdout = open(dbname + "file.txt", 'w+')
    run_codeql_database_create(args, test_db=dbname, db=None, stdout=stdout, lang="csharp")
    stdout.seek(0)
    s = stdout.read()
    stdout.close()
    return s

# Compile and run the program.
# It is expected that 'hello1' is printed to stdout.
s = run_codeql_database_create_stdout(['dotnet run -- hello1 hello2'], 'without-argument-db')

// validate s
```